projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -370,17 +370,7 @@ export class DatatableComponent<TRow = any>
    * totalMessage     [default] = 'total'
    * selectedMessage  [default] = 'selected'
    */
-  @Input() messages: Partial<INgxDatatableConfig['messages']> = {
-    // Message to show when array is presented
-    // but contains no values
```

Comment:
WDYT, shall we move the parameter description up to the code comment, so we don't lose them? IMO, `Message to show when array is presented` is much more descriptive than `emptyMessage`.

Reply:
Updated the interface instead.

I could also copy those texts into the property description here, but then it would be redundant.